packages/manager/apps/dedicated/client/app/account/user/users/sso/details/user-users-sso-details.html

```diff
@@ -0,0 +1,64 @@
+<oui-header heading="{{:: 'user_users_sso_detail_title' | translate }}">
+</oui-header>
+<div>
+    <a
+        class="d-inline-block col-md-12 mb-12"
+        data-ng-href="{{'#/useraccount/users' }}"
+        title="{{ 'user_users_sso_detail_back' | translate }}"
+        style="margin-bottom: 1rem;"
```

Comment:
**question:** Could you please avoid using inline style?

Reply:
inline styles removed